### ISSUE TITLE
Early access to .NET 5 setting keeps reverting

### ISSUE THREAD

**Issue by clement911** [Azure/app-service-linux-docs#64]
We've enabled .NET 5 in Azure app application settings but the settings keeps reverting back to .Net core automatically.

![image](https://user-images.githubusercontent.com/3426504/100851618-bc565000-3485-11eb-8b31-fba81d51a6ab.png)

**Comment by cfdelaune** [comment]
We are having the same issue in a Windows based AppService. Have you found any more information about this issue?

We just made the switch in production over the weekend (1/3/2021) and this is a very strange problem. In our setup we use a Deployment Slot for a staging instance along side production. I am wondering if the use of Deployment Slots has anything to do with this.

**Comment by otaskiran** [comment]
It is already latest version of .Net Core. What is your problem?

**Comment by cfdelaune** [comment]
You are right. I received an error upon publish and when I checked, it had .NET Core selected instead of .NET, so I assumed that was the problem. I have pushed an update without changing anything, it reverted to .NET Core and everything seems to work now.

In [this StackOverflow article](https://stackoverflow.com/questions/64777919/deploy-asp-net-core-5-app-to-azure-app-service), there is NOTE in the accepted answer stating that you "could" switch back to .NET Core since the new runtime has technically been "installed", but it also says you probably should not do that. So, I am still curious if there is a bug that is automatically changing my selection somehow.

I assume, at some point, .NET 5 will be available within an AppService without having to rely on this "Early Access" feature described [here](https://devblogs.microsoft.com/aspnet/announcing-asp-net-core-in-net-5/), but cannot find documentation as to when that might happen. Can you point me in the right direction? When deploying from VS I am still receiving the following warning, but am hopeful that it goes away when .NET 5 is natively available within the AppService.

![20210111 VS Publish Warning](https://user-images.githubusercontent.com/105791/104193585-44693700-53ee-11eb-906c-c65255fbae28.jpg)

**Comment by elvirdolic** [comment]
We have the same issue. We changed the setting in azure portal, after deployment the setting was overridden again. The big issue is that the app still is working with wrong setting and suddenly it stops. In our case it stopped out of office hours and we had a long downtime.

![image](https://user-images.githubusercontent.com/17769458/111118331-edf0b400-8568-11eb-8f9a-fe5abc1a19b1.png)

4 days after the deployment it just crashed ... this was the moment we realized that the .net5 earyl access setting was reverted to asp .net 4.8

![image](https://user-images.githubusercontent.com/17769458/111118369-fb0da300-8568-11eb-9cd6-92f52959c0b8.png)
